shell/platform/windows/window_state.h

```diff
@@ -12,8 +12,16 @@
 namespace flutter {
 struct FlutterWindowsEngine;
 struct FlutterWindowsView;
+struct FlutterWindowsTextureRegistrar;
 }  // namespace flutter
 
+// Opaque reference to the internal texture registrar
+struct FlutterDesktopTextureRegistrar {
```

Comment:
There's no need for the struct at all; you can just use a pointer to the class directly as the ref. See the engine ref for an example of that.

The goal is to eliminate all of these structs.